lib/user_suspensions_exporter.rb

```diff
@@ -0,0 +1,52 @@
+require 'csv'
+
+class UserSuspensionsExporter
+  attr_reader :applications, :export_dir, :users_since_date, :suspensions_since_date, :logger
+
+  def initialize(export_dir, users_since_date, suspensions_since_date, logger = Rails.logger)
+    @export_dir = export_dir
+    @users_since_date = users_since_date
+    @suspensions_since_date = suspensions_since_date
+    @logger = logger
+  end
+
+  def export_suspensions
```

Comment:
Since the class does only one thing it might be nice to make this a `call` method, then the `attr_reader` and `file_path` could be private.

Reply:
Would the `file_path` being private affect the stubbing in the tests?

Reply:
I believe Rspec is able to handle that.